ReducingCrime.Rmd

```diff
@@ -85,7 +96,7 @@ summary(crimeData$prbpris)
 ## Univariate Variable Analysis of Key Variables
 
 ### crmrte: crimes committed per person
-Looking at the histogram of crime per person, the distribution appears to be positively skewed. Applying `log()` on crime shows the histogram to appear normally distributed.
+Looking at the histogram of crime per person, the distribution appears to be positively skewed. Applying `log()` on crime shows the histogram to appear normally distributed. The linear regressions would benefit from this transformation.
```

Comment:
I don't see much value by adding same sentence "The linear regressions would benefit from this transformation" to each key variable analysis. I was happy with earlier text. I don't think during initial EDA we would know whether variable will benefit the model or not. If we can say something about how it could benefit then yes we can add this line otherwise lets leave the original text